src/components/TodoList/TodoList.tsx

```diff
@@ -1,44 +1,104 @@
 import React from 'react';
+import classNames from 'classnames';
+import { TodoForm } from '../TodoForm';
 import './TodoList.scss';
 
-export const TodoList: React.FC = () => (
-  <div className="TodoList">
-    <h2>Todos:</h2>
-
-    <div className="TodoList__list-container">
-      <ul className="TodoList__list">
-        <li className="TodoList__item TodoList__item--unchecked">
-          <label>
-            <input type="checkbox" readOnly />
-            <p>delectus aut autem</p>
-          </label>
-
-          <button
-            className="
-              TodoList__user-button
-              TodoList__user-button--selected
-              button
-            "
-            type="button"
-          >
-            User&nbsp;#1
-          </button>
-        </li>
-
-        <li className="TodoList__item TodoList__item--checked">
-          <label>
-            <input type="checkbox" checked readOnly />
-            <p>distinctio vitae autem nihil ut molestias quo</p>
-          </label>
-
-          <button
-            className="TodoList__user-button button"
-            type="button"
-          >
-            User&nbsp;#2
-          </button>
-        </li>
-      </ul>
-    </div>
-  </div>
-);
+type Props = {
+  todos: Todo[];
+  onUsersId: (userId: number) => void;
+};
+
+type State = {
+  filteredQuery: string,
+  selectedQuery: string,
+};
+
+export class TodoList extends React.Component<Props, State> {
+  state: State = {
+    filteredQuery: '',
+    selectedQuery: '',
+  };
+
+  handleChange = (event: InputOrSelect) => {
+    const { name, value } = event.target;
+
+    this.setState({ [name]: value } as Pick<State, keyof State>);
+  };
+
+  getFilteredTodos = () => {
+    const { todos } = this.props;
+    const query = this.state.filteredQuery.toLowerCase();
+
+    switch (this.state.selectedQuery) {
+      case 'all':
+        return todos
+          .filter(todo => todo.title.toLowerCase().includes(query));
+
+      case 'active':
+        return todos
+          .filter(todo => !todo.completed
+          && todo.title.toLowerCase().includes(query));
```

Comment:
DRY. You can separate this filter in another variable and then use this new variable here